packages/grid/_modules_/grid/hooks/features/export/serializers/excelSerializer.ts

```diff
@@ -0,0 +1,157 @@
+import type * as Excel from 'exceljs';
+import {
+  GridCellParams,
+  GRID_CHECKBOX_SELECTION_COL_DEF,
+  GridStateColDef,
+  GridRowId,
+  GridColDef,
+  GridValueOptionsParams,
+  GridValueFormatterParams,
+  GridApi,
+} from '../../../../models';
+
+const getExcelJs = () => import('exceljs');
+
+const getFormattedValueOptions = (
+  colDef: GridColDef,
+  { id, row, field }: GridValueOptionsParams,
+  api: GridApi,
+) => {
+  // TODO: clean depending on the solution chosen for https://github.com/mui-org/material-ui-x/issues/3806
+  if (!colDef.valueOptions) {
+    return [];
+  }
+  let valueOptionsFormatted;
+  if (typeof colDef.valueOptions === 'function') {
+    valueOptionsFormatted = colDef.valueOptions({ id, row, field });
+  } else {
+    valueOptionsFormatted = colDef.valueOptions;
+  }
+
+  if (colDef.valueFormatter) {
+    valueOptionsFormatted = valueOptionsFormatted.map((option) => {
+      if (typeof option === 'object') {
+        return option;
+      }
+
+      const params: GridValueFormatterParams = { field, api, value: option };
+      return String(colDef.valueFormatter(params));
+    });
+  }
+  return valueOptionsFormatted.map((option) =>
+    typeof option === 'object' ? option.label : option,
+  );
+};
+
+const serialiseRow = (
+  id: GridRowId,
+  columns: GridStateColDef[],
+  getCellParams: (id: GridRowId, field: string) => GridCellParams,
+  api: GridApi,
+) => {
+  const row = {};
+  const dataValidation = {};
+
+  const firstCellParams = getCellParams(id, columns[0].field);
+  const outlineLevel = firstCellParams.rowNode.depth;
+
+  columns.forEach((column) => {
+    const cellParams = getCellParams(id, column.field);
+    switch (cellParams.colDef.type) {
+      case 'singleSelect': {
+        const valueOptions: GridValueOptionsParams = {
+          id: cellParams.id,
+          row: cellParams.row,
+          field: cellParams.field,
+        };
+        const formattedValueOptions = getFormattedValueOptions(
+          cellParams.colDef,
+          valueOptions,
+          api,
+        );
+        dataValidation[column.field] = {
+          type: 'list',
+          allowBlank: true,
+          formulae: [formattedValueOptions.map((x) => `"${x}"`)],
```

Comment:
```suggestion
          formula: [formattedValueOptions.map((x) => `"${x}"`)],
```

Reply:
I don't know why they added this "e", but it is an exceljs naming [formattedValue](https://github.com/exceljs/exceljs#data-validations)

Reply:
Fun fact: "formulae" is the plural of "formula": https://dictionary.cambridge.org/us/dictionary/english/formulae

Makes sense this name since it's an array.

Reply:
I thought it was kind of abbreviation for "formula excel" 😂